Softeq.XToolkit.WhiteLabel.Droid/ImagePicker/ImagePickerActivity.cs

```diff
@@ -0,0 +1,301 @@
+﻿
+using System;
+using System.Globalization;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Android.App;
+using Android.Content;
+using Android.Database;
+using Android.Graphics;
+using Android.Media;
+using Android.OS;
+using Android.Provider;
+using Android.Runtime;
+using Android.Support.V4.Content;
+using Android.Widget;
+using Plugin.CurrentActivity;
+using ImageOrientation = Android.Media.Orientation;
+
+namespace Softeq.XToolkit.WhiteLabel.Droid.ImagePicker
+{
+    [Activity(Label = "ImagePickerActivity")]
+    internal class ImagePickerActivity : Activity
```

Comment:
need to discuss, that we really want to support our activity or use media plugin activity